workflow/kfdrc-rnaseq-workflow.cwl

```diff
@@ -0,0 +1,379 @@
+  5) Suggested inputs are:
+
+      - `FusionGenome`: [GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz](https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB/__genome_libs_StarFv1.3/GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz)
+      - `gtf_anno`: gencode.v27.primary_assembly.annotation.gtf, location: ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_27/gencode.v27.primary_assembly.annotation.gtf.gz, will need to unzip
+      - `RNAseQC_GTF`: gencode.v27.primary_assembly.RNAseQC.gtf, built using `gtf_anno` and following build instructions [here](https://github.com/broadinstitute/rnaseqc#usage)
+      - `RSEMgenome`: RSEM_GENCODE27.tar.gz, built using the `reference_fasta` and `gtf_anno`, following `GENCODE` instructions from [here](https://deweylab.github.io/RSEM/README.html), then creating a tar ball of the results.
+      - `STARgenome`: STAR_GENCODE27.tar.gz, created using the star_genomegenerate.cwl tool, using the `reference_fasta`, `gtf_anno`, and setting `sjdbOverhang` to 100
+      - `reference_fasta`: [GRCh38.primary_assembly.genome.fa](ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_27/GRCh38.primary_assembly.genome.fa.gz), will need to unzip 
+      - `kallisto_idx`: gencode.v27.kallisto.index, built from gencode 27 trascript fasta: ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_27/gencode.v27.transcripts.fa.gz, following instructions from [here](https://pachterlab.github.io/kallisto/manual)
+
+  ### Outputs:
+  ```yaml
+  outputs:
+    cutadapt_stats: {type: File, outputSource: cutadapt/cutadapt_stats} # only if adapter supplied
+    STAR_transcriptome_bam: {type: File, outputSource: star/transcriptome_bam_out}
+    STAR_sorted_genomic_bam: {type: File, outputSource: samtools_sort/sorted_bam}
+    STAR_sorted_genomic_bai: {type: File, outputSource: samtools_sort/sorted_bai}
+    STAR_chimeric_bam_out: {type: File, outputSource: samtools_sort/chimeric_bam_out}
+    STAR_chimeric_junctions: {type: File, outputSource: star_fusion/chimeric_junction_compressed}
+    STAR_gene_count: {type: File, outputSource: star/gene_counts}
+    STAR_junctions_out: {type: File, outputSource: star/junctions_out}
+    STAR_final_log: {type: File, outputSource: star/log_final_out}
+    STAR-Fusion_results: {type: File, outputSource: star_fusion/abridged_coding}
+    arriba_fusion_results: {type: File, outputSource: arriba_fusion/arriba_fusions}
+    arriba_fusion_viz: {type: File, outputSource: arriba_fusion/arriba_pdf}
+    RSEM_isoform: {type: File, outputSource: rsem/isoform_out}
+    RSEM_gene: {type: File, outputSource: rsem/gene_out}
+    RNASeQC_Metrics: {type: File, outputSource: rna_seqc/Metrics}
+    RNASeQC_counts: {type: File, outputSource: supplemental/RNASeQC_counts} # contains gene tpm, gene read, and exon counts
+    kallisto_Abundance: {type: File, outputSource: kallisto/abundance_out}
+  ```
+
+  # D3b annoFuse Workflow
```

Comment:
The downside of importing the README as-is, that the repo itself may have more than the app. Please remove this part from the RNAseq workflow `doc`

Reply:
Removed.